packages/block-library/src/media-text/media-container.native.js

```diff
@@ -62,17 +61,11 @@ class MediaContainer extends Component {
 	}
 
 	componentDidMount() {
-		const { mediaId, mediaUrl, onMediaUpdate, mediaType } = this.props;
-
-		if ( mediaId && mediaUrl && ! isURL( mediaUrl ) ) {
-			if ( mediaUrl.indexOf( 'file:' ) === 0 && mediaType === MEDIA_TYPE_IMAGE ) {
-				// We don't want to call this for video because it is starting a media upload for the cover url
-				requestMediaImport( mediaUrl, ( id, url ) => {
-					if ( url ) {
-						onMediaUpdate( { id, url } );
-					}
-				} );
-			}
+		const { mediaId, mediaUrl, mediaType } = this.props;
+
+		// Make sure we mark any temporary images as failed if they failed while
+		// the editor wasn't open
+		if ( mediaId && mediaUrl && mediaUrl.indexOf( 'file:' ) === 0 && mediaType === MEDIA_TYPE_IMAGE ) {
```

Comment:
We should remove `mediaType === MEDIA_TYPE_IMAGE` it was a condition for `requestMediaImport `